client/components/account-page.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import {Link} from 'react-router-dom'
+
+export const AccountPage = () => {
+    console.log('hello');
+    return (
+        <div>
+            <h1>Account Page</h1>
+            <div>
+                <Link to="/cart">Cart</Link>
+            </div>
+            <div>
+                <Link to="/order-history">Order History</Link>
```

Comment:
Instead of a link to Order History, we'll have a map that lists the order history on the account page.

Routes to this page will eager load the orders made by this user.

Import Orders reducer
Add in a map function that will populate a list of orders based on the user Id